src/hooks.tsx

```diff
@@ -153,12 +155,17 @@ export const useWindowSize = (): {
         isClientSide() ? window.screen.availWidth : 800
     );
 
+    const [visualHeight, setVisualHeight] = React.useState<number>(
+        isClientSide() ? window.visualViewport?.height || window.screen.availWidth : 1200
```

Comment:
availWidth? Shouldn't this be availHeight?

Reply:
oh! good catch. Thanks